notifier/notifier.go

```diff
@@ -126,14 +103,45 @@ func New(s *storage.Storage, concurrency int, logger *log.Logger, dwnlURL string
 
 // Start starts the Notifier loop and instruments the worker goroutines that
 // perform the actual notify requests.
```

Comment:
Please document what `cfg` is or even better, rename to something more informative like `backendCfg`.